coalib/bearlib/aspects/Smell.py

```diff
@@ -324,3 +552,44 @@ class docs:
         'This the maximum number of embedded branches or embedded loops'
         ' allowed.',
         (6,), default=6)
+
+
+@Complexity.subaspect
+class MaintainabilityIndex:
+    """
+    Software metric which measure how maintainable is a program.
+
+    The `maintainablity index
+    <www.projectcodemeter.com/cost_estimation/help/GL_maintainability.htm>`_
+    is always in the range 0-100 and is ranked
+    as follow:
+
+    * A `MI` in the range 0-9 maps to a code extremely difficult to maintain.
+    * A `MI` in the range 10-19 maps to a maintainable code.
+    * A `MI` in the range 20-100 maps to a code highly maintainable.
+    """
+    class docs:
+        example = """
+        '''
+        The maintainability index of our `linter` class along with
+        the `_create_linter` and `_prepare_options` functions found `Here
+        <https://github.com/coala/coala/blob/master/coalib/bearlib/abstractions
```

Comment:
no particular reason it is just the link to linter.py where _create_linter and _prepare_options are found